release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,18 @@ uid: DCP_change_log
 
 The DataMiner Cloud Platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 13 January 2023 - Enhancement - Auditing of protocol and script deployments [ID_35392]
+
+The *Audit* page of the DCP Admin app will now also contain records for protocol and script deployments, while previously it only showed records for DxM artifact deployments.
+
+#### 12 January 2023 - Fix - CloudGateway 2.10.2 - CloudGateway not starting if DataMiner uses HTTP [ID_35362]
+
+When DataMiner was configured to use HTTP, prior to version 2.10.2, CloudGateway would fail to start up, and it would throw the following exception:
```

Comment:
maybe **could** instead of **would**, because it was only in some recent versions of CloudGateway (didn't check particularly which ones)

Reply:
i was under the impression it would cause it to always fail but i'm ok to change it to Could to be on safe side and make it less critical fix

Reply:
yes but not in 'all' version before 2.10.2, back in the days cloudgateway has been http only so i know it has been working until some point in time

Reply:
extra feedback from discussion with Jordy : Can you add a small description from where deployments can come from?
Connector artifacts can be deployed via Catalog, script artifacts can be deployed via Github CI CD pipeline. Dxm deployments can be done via the admin app.

Reply:
comment ^^ intended for ID 35392